mage_ai/data_cleaner/transformer_actions/column.py

```diff
@@ -67,6 +74,18 @@ def impute(df, action, **kwargs):
 
     if strategy == ImputationStrategy.AVERAGE:
         df[columns] = df[columns].fillna(df[columns].astype(float).mean(axis=0))
+    elif strategy == ImputationStrategy.CONSTANT:
```

Comment:
For constant strategy, current behavior is to use the value from action_options. Let's make sure that logic still works. If the value is not None, let's use the value. Otherwise, use the default constant.

Reply:
Added this behavior + unit test to confirm